plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -299,6 +455,39 @@ private Optional<DataFileInfo> rewriteParquetFile(Path path, ImmutableLongBitmap
         return Optional.of(fileWriter.getDataFileInfo());
     }
 
+    private void storeCdfEntries(Page page, int[] deleted, int deletedCount, FileDeletion deletion)
+    {
+        if (cdfEnabled) {
+            if (cdfPageSink == null) {
+                cdfPageSink = cdfPageSinkSupplier.get();
+            }
+            Page cdfPage = page.getPositions(deleted, 0, deletedCount);
+            Block[] outputBlocks = new Block[nonSynthesizedColumns.size() + 1];
+            int cdfPageIndex = 0;
+            int partitionIndex = 0;
+            List<String> partitionValues = deletion.partitionValues;
+            for (int i = 0; i < nonSynthesizedColumns.size(); i++) {
+                if (nonSynthesizedColumns.get(i).getColumnType() == REGULAR) {
+                    outputBlocks[i] = cdfPage.getBlock(cdfPageIndex);
+                    cdfPageIndex++;
+                }
+                else {
+                    outputBlocks[i] = RunLengthEncodedBlock.create(nativeValueToBlock(
+                            nonSynthesizedColumns.get(i).getType(),
+                            deserializePartitionValue(
+                                    nonSynthesizedColumns.get(i),
+                                    Optional.of(partitionValues.get(partitionIndex)))
+                    ), cdfPage.getPositionCount());
```

Comment:
can you please verify you got a checkstyle violation for this `)` brace?

Reply:
I don't, the build is green

Reply:
we have this check
https://github.com/airlift/airbase/blob/16557bf5436bee8b49f6352e4575ab5852de3448/airbase-policy/src/main/resources/checkstyle/airbase-checks.xml#L36-L37

i don't know why it checks for trailing content after the brace though

please put the `)` on the prev line anyway

Reply:
> i don't know why it checks for trailing content after the brace though

i understand now.
these additional conditions help differentiate between Java code and String literal content (eg SQL queries)